public/swagger/User.json

```diff
@@ -17,6 +17,9 @@
     },
     "hasGithubAuth": {
       "type": "boolean"
+    },
+    "signedInAt": {
+      "type": "date-time"
```

Comment:
hmm... `signedInLastAt`? `lastSignIn` ?

Reply:
That's the name of the existing field, already surfaced through the API for admins. It's just newly available for others.

Reply:
those `adminerers` and their awful variable names